hypothesis-python/src/hypothesis/extra/array_api.py

```diff
@@ -974,3 +974,21 @@ def mock_finfo(dtype: DataType) -> FloatInfo:
         any=np.any,
         all=np.all,
     )
+
+
+def installed_array_modules() -> Dict[str, Any]:
+    """Returns list of array modules
```

Comment:
🤨

But also, let's have this live in `tests/array_api/common.py` for now - we can move it over to the installed library when we have a user-facing case for it.  That in turn means we don't need release notes, etc.